packages/target-electron/src/deltachat/webxdc.ts

```diff
@@ -731,13 +756,18 @@ export default class DCWebxdc {
           }
           const messageWithMap = await this.rpc.getMessage(accountId, msgId)
           if (messageWithMap && messageWithMap.webxdcInfo) {
-            openWebxdc(evt, msgId, {
-              accountId,
-              displayname: '',
-              chatName,
-              webxdcInfo: messageWithMap.webxdcInfo,
-              href: '',
-            })
+            openWebxdc(
+              evt,
+              msgId,
+              {
+                accountId,
+                displayname: '',
+                chatName,
+                webxdcInfo: messageWithMap.webxdcInfo,
+                href: '',
+              },
+              true
+            )
```

Comment:
To be honest I don't like it when we pass around booleans. Why not just pass the default bounds in here instead of true?